core/put/source/dev_ui.h

```diff
@@ -64,8 +64,9 @@ namespace put
             };
         }
         typedef e_ui_shader::ui_shader_t ui_shader;
-
-        bool        init();
+        ImGuiStyle& default_pmtech_style();
+        bool        init(ImGuiStyle& style = default_pmtech_style());
+        bool        init(ImGuiStyle& style);
```

Comment:
We don't need 2 init functions. just `bool        init(ImGuiStyle& style = default_pmtech_style());`